scripts/deploy.sh

```diff
@@ -9,10 +9,12 @@ jlpm run build:prod
 npm publish --access public # token is configured in ~/.npmrc; requires OTP
 
 git clean -dfx
-python setup.py sdist bdist_wheel
-twine upload -u __token__ -p $(cat ~/.tokens/pypi_jupyter_bokeh) dist/jupyter_bokeh-*.tar.gz
+hatch build
+hatch publish -u __token__ -a $(cat ~/.tokens/pypi_jupyter_bokeh)
+
+VERSION=$(find dist -name "*.whl" -exec basename {} \; | cut -d- -f2)
```

Comment:
Can be updated with `hatch version`

Reply:
Can you make the corresponding update in Panel?

Reply:
Panel does not use hatch

Reply:
Duh!